src/Routes/Routes.jsx

```diff
@@ -0,0 +1,11 @@
+import { Route } from "react-router-dom";
+import { StartPage } from "../Pages/Startpage";
+import { AboutUsPage } from "../Pages/AboutUsPage";
+
+export const routes = (
+  <>
+    {/* ------- Page Routes ------- */}
+    <Route path="/" element={<StartPage />} />
+    <Route path="/aboutus-page" element={<AboutUsPage />} />
+  </>
+);
```

Comment:
Great set up with the routes, makes the app.jsx look nice and easy to follow the structure.